e2e-tests/portal/cypress/e2e/journeys/checker/cannot-submit-a-deal-with-newly-issued-facilities-that-have-cover-start-dates-before-min-submission-date/cannot-submit-a-deal-with-newly-issued-facilities-that-have-cover-start-dates-before-min-submission-date.spec.js

```diff
@@ -1,29 +1,24 @@
 const pages = require('../../../pages');
 const relative = require('../../../relativeURL');
-const theDeal = require('./deal');
 const MOCK_USERS = require('../../../../../../e2e-fixtures');
 
-const { ADMIN, BANK1_MAKER1, BANK1_CHECKER1 } = MOCK_USERS;
+const { ADMIN, BANK1_CHECKER1 } = MOCK_USERS;
 
 // DTFS2-2839
 context(
   'Checker tries to submit a deal that has changed/newly issued facilities (in `Ready for check` status) with cover start dates that are before MIN submission date',
   () => {
-    let deal;
     let dealId;
 
     before(() => {
       cy.deleteDeals(ADMIN);
 
-      cy.insertOneDeal(theDeal, BANK1_MAKER1).then((insertedDeal) => {
-        deal = insertedDeal;
-        dealId = deal._id;
-      });
+      cy.createBssDeal({ readyForCheck: true });
     });
 
     it('should throw error and not submit or redirect', () => {
       cy.login(BANK1_CHECKER1);
-      pages.contract.visit(deal);
+      pages.dashboardDeals.rowIndex.link(1).click();
```

Comment:
We can update the dashboard to have a default row index

Reply:
I've updated this. I do however think we should create a `clickFirstDashboardDealLink` command or something like that so it's clearer and doesn't require importing in individual tests/commands